devtools/maven/src/main/java/io/quarkus/maven/CreateUtils.java

```diff
@@ -81,6 +82,14 @@ static QuarkusPlatformDescriptor resolvePlatformDescriptor(final String bomGroup
             version = resolvePluginInfo(CreateUtils.class).getVersion();
         }
 
+        if((bomGroupId==null && bomArtifactId==null && bomVersion==null) || bomArtifactId=="quarkus-universe-bom"){
```

Comment:
Use `.equals()` instead of `==`

Reply:
Also it would be nice to invert the comparison, to make sure that a NPE isn't accidentally thrown: 

`"quarkus-universe-bom".equals(bomArtifactId)`

Reply:
okk